depthmapXcli/commandlineparser.cpp

```diff
@@ -25,11 +25,12 @@
 using namespace depthmapX;
 
 void CommandLineParser::printHelp(){
-    std::cout << "Usage: depthmapXcli -m <mode> -f <filename> -o <output file> [-s] [mode options]\n"
+    std::cout << "Usage: depthmapXcli -m <mode> -f <filename> -o <output file> [-s] [-t] [-p] [mode options]\n"
```

Comment:
If adding `-t` to the help output, it should be `[-t <times.csv>]`. This option requires a file argument.